packages/api-gateway/__tests__/geolocation.test.js

```diff
@@ -0,0 +1,35 @@
+const boom = require("@hapi/boom");
+const pluginLoader = require("fastify-plugin");
+const { fastify } = require("../src/drivers/http/server");
+
+test("given an object of type place, when it is required to write to the DB, it returns a status code 200", async () => {
+  const places = await fastify.inject({
+    method: "POST",
+    url: "geolocation/place",
+    body: {
+      lon: -14.16546,
+      lat: 40.16546,
+      country: "México",
+      state: "Monterey",
+      city: "Cidad de Monterrey",
+      zipcode: "165466",
+      streetAddress: "av. pino suarez",
```

Comment:
@Edilberto-Vazquez what do you think if instead of hardcoding the body data which is an antipattern we use [Faker](https://github.com/faker-js/faker) to create these fake values?